package.json

```diff
@@ -10,6 +10,20 @@
     "prettier": "prettier --write './**'",
     "lint": "eslint src"
   },
+  "moduleNameMapper": {
```

Comment:
Should this be inside of the `"jest"` property?

Reply:
Yeah, my mistake actually, was playing around with the nameMapper